.github/yarn-install/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Yarn install
+
+runs:
+  using: "composite"
+  steps:
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+      shell: bash
+    - uses: actions/cache@v2
+      if: github.actor != 'dependabot[bot]'
```

Comment:
Gir dette mening? 
Bruker ikke cache/lagrer cache for disse jobbene for github. Mest for å ikke fylle opp cachen av dependabot sine byggen.